MetaMorpheus/TaskLayer/MetaMorpheusTask.cs

```diff
@@ -621,6 +621,44 @@ protected List<Protein> LoadProteins(string taskId, List<DbForTask> dbFilenameLi
             {
                 Warn("Warning: " + emptyProteinEntries + " empty protein entries ignored");
             }
+
+            
+
+            if (!proteinList.Any(p => p.IsDecoy))
+            {
+                Status("Done loading proteins", new List<string> { taskId });
+                return proteinList;
+            }
+
+            // Sanitize the decoys
```

Comment:
i need to understand this and how it works with multiple proteases.

Reply:
add a comment and generate a new issue that this does not work for multi-protease